mantidproject/mantid
# Show StartX and EndX properties in fit browser of EngDiff UI

**Description of work.**

Made StartX and EndX properties of Fit browser visible in EngDiff UI fitting tab

**To test:**

(1) Open EngDiffUI
(2) Go to fitting tab
(3) Load data (instructions for making data can be found here https://developer.mantidproject.org/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.html)
(4) Plot the data
(5) Click Fit in the plot toolbar to open the fit browser
(6) Should see the StartX and EndX boxes

![image](https://user-images.githubusercontent.com/55979119/134534184-b71f896f-754a-49d9-8c57-2221b3264165.png)


*There is no associated issue.*






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
